### ISSUE TITLE
Questions about WZG/update_ML/run/Eval.py

### ISSUE THREAD

**Issue by JinyongShin** [yeopjjang/WZG#1]
I think this code is for evaluation. If I am right, why epoch, learning rate, and optimizer is defined in the code? 

I think you can remove them and just import and load your saved model. Also I suggest you add 'model.eval()' in your code. Thank you.
